app/src/main/java/org/greenstand/android/TreeTracker/database/dao/TreeDao.kt

```diff
@@ -0,0 +1,39 @@
+package org.greenstand.android.TreeTracker.database.dao
+
+import androidx.room.Dao
+import androidx.room.Query
+import androidx.room.Transaction
+
+@Dao
+interface TreeDao {
+
+    @Transaction
+    @Query(
+        "SELECT " +
```

Comment:
Note: for all these multi line strings, we can use triple quotes to avoid using `+` at the end of every line.
For example
`
"""
SELECT
tree_id as tree_id,
other stuff
more stuff
""".trimIndent()
`

Reply:
Let's make this coding style fix before we merge, good eye to that stuff @Elforama much appreciate it!

Reply:
updated !